docs/data/material/getting-started/templates/dashboard/components/Navbar.js

```diff
@@ -76,7 +100,7 @@ function Navbar({ mode, toggleColorMode }) {
         >
           <NavbarBreadcrumbs />
           <Stack direction="row" gap={1}>
-            <MenuButton showBadge>
+            <MenuButton showBadge ariaLabel="Notification menu button">
```

Comment:
```suggestion
            <MenuButton showBadge ariaLabel="Open notifications">
```

No need to say "menu button" or describe the component via aria-labels — other attributes will already do that via VoiceOver and such.